examples/contact-sensor-app/nxp/zap-lit/contact-sensor-app.matter

```diff
@@ -1747,7 +1751,7 @@ endpoint 0 {
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 0;
+    ram      attribute featureMap default = 1;
```

Comment:
Please ensure your sample passes TC_DeviceBasicComposition.py and TC_DeviceConformance.py tests before merging.

Reply:
Is the contact sensor app actually using the ACL extension?

Reply:
We have tested the respective tests and all pass.
Also, I have disabled the ACL extension.
Thank you